runtime/verbose/verbose.c

```diff
@@ -845,7 +846,8 @@ initializeVerbosegclogFromOptions(J9JavaVM* vm, char* vbgclogBuffer)
 
 	/* Now look for fileCount */
 	vbgclogBufferPtr += strlen(vbgclogBufferPtr) + 1;
-	if (*vbgclogBufferPtr) {
+	/* if vbgclogBufferPtr point to behind bufferSize, there is no fileCount option */
+	if ((vbgclogBufferPtr < (vbgclogBuffer + bufferSize)) && *vbgclogBufferPtr) {
```

Comment:
Would you please replace `*vbgclogBufferPtr` to `(0 != *vbgclogBufferPtr)` everywhere in this function?

Reply:
... or `('\0' != *vbgclogBufferPtr)` if it would be more correct

Reply:
done

Reply:
There is one more place in this function where `(*vbgclogBufferPtr)` check is used (a few lines above). Would you please change it as well for consistency?

Reply:
done